akka-docs/src/main/paradox/coordinated-shutdown.md

```diff
@@ -0,0 +1,106 @@
+# Coordinated Shutdown
+
+Under normal conditions when `ActorSystem` is terminated or the JVM process is shut down certain
+actors and services will be stopped in a specific order. 
+
+This is handled by an extension named `CoordinatedShutdown`. It will run the registered tasks
+during the shutdown process. The order of the shutdown phases is defined in configuration `akka.coordinated-shutdown.phases`.
```

Comment:
I changed the wording of above paragraphs, but otherwise this page is a verbatim copy of previous content